src/miscellaneous.css

```diff
@@ -77,8 +77,9 @@
 
 .loading--dark {
   color: var(--white);
-  border-color: var(--lighten25);
-  border-top-color: currentColor;
```

Comment:
border-top-color is already defined on line 74 - now that we don't have any overrides to it, we don't need to redeclare it.